packages/atomic/cypress/integration/result-list/result-components/result-multi-value-text-selectors.ts

```diff
@@ -0,0 +1,20 @@
+import {ResultListSelectors} from '../result-list-selectors';
+
+export const resultMultiValueTextComponent = 'atomic-result-multi-value-text';
+
+export const ResultMultiValueTextSelectors = {
+  value: () =>
+    ResultMultiValueTextSelectors.firstInResult()
```

Comment:
why dont we have shadow selector = ResultMultiValueTextSelectors.firstInResult().shadow(), probably easier to read